### DISCUSSION TITLE
Why my Android Shows 32GB only?

### DISCUSSION THREAD

**Discussion by EliranTurgeman** [tesla-android/issue-tracker#235] [Q&A]
Hi 
I installed the latest version, as documented at
Install guide (2023.7.1) - Single image method - https://teslaandroid.com/install-guide

I used a 128GB SDCard
But my Android shows that I have only 32GB on the system. 
![CleanShot 2023-03-09 at 10 40 52 UTC](https://user-images.githubusercontent.com/98872315/223999952-59fdf899-3e60-4a4b-8ad8-a1949f517bdd.png)

What did I miss?

**Comment by mikegapinski** [comment]
You did not miss anything, the data partition does not expand automatically after flashing yet. I have this implemented already and will include in the next build.

You can use parted or any other partition manager to expand the last partition on the sdcard manually

**Reply by EliranTurgeman** [reply]
Thanks you. Keep the good work 👌🏽

**Reply by gauner1986** [reply]
It's probably worth mentioning that resizing needs to happen before booting from the SD card for the first time. During first boot the userdata partition is obviously converted to a non standard filesystem (yaffs2?) which cannot be read by default by most linux distributions. I guess also it can't be resized anyway.

**Reply by mikegapinski** [reply]
The automatic expand of the user partition is implemented and read for the next version ;)
